website/content/en/docs/advanced-topics/custom-bundle-validation.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Custom Bundle Validation
+weight: 80
+---
+
+## Summary
+
+Operator Authors can now use "external" validators with the
+`operator-sdk bundle validate` command by using the
+`--alpha-select-external` flag. This feature enable Operator authors,
+users, and registry pipelines to use custom validators.
+
+## Usage
+
+External validators can be used by specifying a list of local filepaths to
+executables using colons as path separators:
+
+```sh
+operator-sdk bundle validate --alpha-select-external path/validator1:path/validator2
+```
+
+## Writing a Custom Validator
+
+For a validator to work with `operator-sdk bundle validate` each of the files must:
+1. Be executable with appropriate permissions
+1. Return JSON to STDOUT in the [ManifestResult][manifest_result] format.
+
+### Custom Validator from Scratch
+
+Using the `errors` package from `operator-framework/api`, we can start
+by validating the correct number of arguments and marshaling a
+`ManifestResult` into STDOUT.
+
+```go
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/operator-framework/api/pkg/validation/errors"
+)
+
+func main() {
+
+	// we expect a single argument which is the bundle root.
+	// usage: validator-poc <bundle root>
+	if len(os.Args) < 2 {
+		fmt.Printf("usage: %s <bundle root>\n", os.Args[0])
+		os.Exit(1)
+	}
+
+	var validatorErrors []errors.Error
+	var validatorWarnings []errors.Error
+	result := errors.ManifestResult{
+		Name:     "Always Green Example",
+		Errors:   validatorErrors,
+		Warnings: validatorWarnings,
+	}
+	prettyJSON, err := json.MarshalIndent(result, "", "    ")
+	if err != nil {
+		fmt.Println("Invalid json")
+		os.Exit(1)
+	}
+	fmt.Printf("%s\n", string(prettyJSON))
+}
+```
+
+These custom validators can be written in any language. When executed on
```

Comment:
Would it make sense to move the sentence, "These custom validators can be written in any language" up to the _Writing Custom Validators_ section that introduces the topic? It feels like you are burying the lede here and this should be emphasized.